public/app/plugins/datasource/elasticsearch/datasource.test.ts

```diff
@@ -865,7 +865,7 @@ describe('ElasticDatasource', function(this: any) {
     expect((interpolatedQuery.bucketAggs![0] as Filters).settings!.filters![0].query).toBe('resolvedVariable');
   });
 
-  it('should correctly handle empty query strings', () => {
+  it.only('should correctly handle empty query strings', () => {
```

Comment:
the pre-commit hook should have caught this one

Reply:
yeah, I `--no-verify`'ed it just to push it and pick this up later, there are a few other things I want to add to this commit.